wire/internal/wire/wire_test.go

```diff
@@ -542,3 +546,23 @@ func (test *testCase) materialize(gopath string) error {
 	}
 	return nil
 }
+
+// readFile reads data from path. It uses io.Scanner to read the lines of
+// the file, ensuring that they are separated only by "\n".
+// Otherwise, on Windows, file lines that end with "\r\n" cause spurious
+// diffs.
+func readFile(path string) ([]byte, error) {
```

Comment:
`readTextFile` seems a more apt name for this function, as it is interpreting the bytes rather than just verbatim copying.